src/sentry/web/forms/accounts.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, request=None, *args, **kwargs):
             self.fields["username"].label = capfirst(self.username_field.verbose_name)
 
     def clean_username(self):
-        value = (self.cleaned_data.get("username") or "").strip()
+        value = (self.cleaned_data.get("username") or "").strip(" \n\t\r\0")
```

Comment:
Ah yes, this is a good place here.

We should use this cleaned value then in the rate limiter. It was probably just an oversight, or copy/pasted from auth_login where.

Reply:
OK so the reason we don't use the AuthenticationForm username for the rate limiting is because we do the rate limiting before we validate the form. The clean() function seems to do the rate limiting as well so I think we might be doing double checks too.